client/src/pages/SearchResultsUser.js

```diff
@@ -11,6 +12,24 @@ function SearchResultsUser() {
   }
 
   const query = useQuery().get("query");
+  componentDidMount() {
+    fetch("https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed")
+      .then(res => res.json())
+      .then(
+        (result) => {
+          this.setState({
+            isLoaded: true,
+            items: result.items
+          });
+        },
+        (error) => {
+          this.setState({
```

Comment:
Change this to also be a state such as `const [error, setError] = useState(null);`. If you look at the PR I opened for the get calls in Home you'll see that I've set it up similarly, and after I merge, you can pull and use the `ErrorMessage` component I whipped up to display that if you get nothing back.